tests/controllers/qos_controller_test.cpp

```diff
@@ -1,17 +1,21 @@
-#include <gtest/gtest.h>
+#include <list>
 
-#include <mesos/resources.hpp>
+#include "gtest/gtest.h"
```

Comment:
To have "at least any" lint, we took Google's C++ Linter
https://google-styleguide.googlecode.com/svn/trunk/cpplint/cpplint.py

It returns error, when C-style system header (< .h>/< .hpp>) is used after C++-style system header (eg. < list>). We didn't wanted to change it, and just went along with it.